### ISSUE TITLE
jumpwin1 vm extension fails when calling Azure PowerShell module

### ISSUE THREAD

**Issue by doherty100** [Azure-Samples/azuresandbox#22]
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is for a: (mark with an `x`)
```
- [ X ] bug report -> please search issues before submitting
- [ ] feature request
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
> Apply configuratino for terraform-azurerm-vnet-app. Note error when attempting to provision vm extension for jumpwin1.

### Any log messages given by the failure
> Terraform reports a non-zero exit code for vm extension.

### Expected/desired behavior
> there should be no errors

### OS and Version?
> N/A

### Versions
> v2.8.11

### Mention any other details that might be useful

> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

**Comment by doherty100** [comment]
Powershell Az module is installed using chocolatey. Module appears to be installed, but `Get-Module -Name Az -ListAvailable` returns null. Moving module installation from Chocolatey to script based approach to see if this resolves the issue.

**Comment by doherty100** [comment]
Fix will be in next release.

**Comment by doherty100** [comment]
Fixed in release 2.8.12
